phoenix-core/src/main/java/org/apache/phoenix/compile/QueryPlan.java

```diff
@@ -92,4 +94,17 @@
     public boolean useRoundRobinIterator() throws SQLException;
 
     <T> T accept(QueryPlanVisitor<T> visitor);
+
+    /**
+     * <pre>
+     * Get the actual OrderBys of this queryPlan, which may be different from {@link #getOrderBy()},
+     * because {@link #getOrderBy()} is only the compiled result of {@link SelectStatement}.
+     * The return type is List because we can get multiple OrderBys for the query result of {@link SortMergeJoinPlan},
+     * eg. for the sql:
+     * SELECT  * FROM T1 JOIN T2 ON T1.a = T2.a and T1.b = T2.b
+     * The result of the sort-merge-join is sorted on (T1.a, T1.b) and (T2.a, T2.b) at the same time.
+     * </pre>
+     * @return
+     */
+    public List<OrderBy> getActualOutputOrderBys() ;
```

Comment:
There would only be multiple OrderBys in the case of a join, right?

Reply:
Also,  maybe we can come up with a better API name like ``getPhysicalOrderBy`` or ``getFinalOrderBy``, WDYT?

Reply:
Yes,now there would only be multiple OrderBys for SortMergeJoin.

Yes,I think the name `getFinalOrderBy` is OK , I named it as 'getActualOutputOrderBys' because
the equivalent name in Spark SQL's SparkPlan is `outputOrdering`, FYI.

Reply:
``getOutputOrdering`` also works.

Reply:
@twdsilva , thank you for your opinion, in order to in correspondence with the existing name `getOrderBy()`  in `QueryPlan` class, can we named it as `getOutputOrderBy` ?

Reply:
Sure that works